lib/baseClasses/HttpError.js

```diff
@@ -29,10 +29,11 @@ function HttpError() {
     var args = parsed.args;
     var options = parsed.options || {};
 
-    // inherit from WError, call super first before doing anything else!  WError
+    // inherit from WError, call super first before doing anything else! WError
     // will handle calling captureStackTrace, using arguments.callee to
     // eliminate unnecessary stack frames.
     WError.apply(self, args);
+    Error.captureStackTrace(self, HttpError);
```

Comment:
The underlying API changed - check out the linked issue in the PR for more info (I filed an issue over with VError).